packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -48,6 +48,19 @@ const BoxDecoration _kCupertinoDialogBackFill = const BoxDecoration(
   color: const Color(0x77FFFFFFF),
 );
 
+const double _kEdgePadding = 20.0;
+
+const double _kButtonHeight = 45.0;
+
+// This color isn't correct. Instead, we should carve a hole in the dialog and
+// show more of the background.
```

Comment:
please make this a TODO, as in `// TODO(xster): This color...`

Reply:
I didn't add this comment.